sklearn/cluster/_kmeans.py

```diff
@@ -1513,6 +1516,17 @@ class MiniBatchKMeans(KMeans):
         partition (if compute_labels is set to True). The inertia is
         defined as the sum of square distances of samples to their nearest
         neighbor.
+        
+    Counts_ : array like, shape (k,)
+        An array of samples that can be randomly reassigned if the existing
```

Comment:
This description is not clear. Isn't it just the number of sample in each cluster, weighted by sample weights ?